configs/openmrs_config/concepts/physical_examination.csv

```diff
@@ -1,20 +1,20 @@
 Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:km,Short name:en,Short name:km,Description:en,Description:fr,Data class,Data type,Answers,Members,_version:1,_order:1003
-123919AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,,Ulcer,,Ulcer,,,,Misc,N/A,,,,
+123919AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,,Ulcer,ដំបៅ,Ulcer,ដំបៅ,,,Misc,N/A,,,,
 871389dc-a412-40ba-aa6d-d12497065ce0,,,Amputation,កាត់ជើង,Amputation,កាត់ជើង,,,Misc,N/A,,,,
-735dc19f-78f0-4f0f-861b-e2b4328521b4,,,AKA,,AKA,,,,Misc,N/A,,,,
-bb9a023b-a077-4aad-baac-021c8ed1f626,,,BKA,,BKA,,,,Misc,N/A,,,,
-81318009-e9fb-4b8a-9bf6-17082cefd19e,,,Symptom of PN-Pain,,Symptom of PN-Pain,,,,Misc,N/A,,,,
-be0c7342-12da-48c1-a247-89379f80e18d,,,Symptom of PN-Pins and Needle,,Symptom of PN-Pins and Needle,,,,Misc,N/A,,,,
-4ff6c3ea-2be8-4aac-b371-faa3bb450255,,,Symptom of PN-Numbness,,Symptom of PN-Numbness,,,,Misc,N/A,,,,
-8c001d6a-9525-4008-8d06-eebc713f5d8c,,,Symptom of PN-Burning,,Symptom of PN-Burning,,,,Misc,N/A,,,,
-0571152b-1f18-4603-9c86-d77883525e61,,,Symptom PAD-Yes,,Symptom PAD-Yes,,,,Misc,N/A,,,,
-c9cfaaf9-0f14-4805-a151-b0544cbd8870,,,Symptom of PAD-No,,Symptom of PAD-No,,,,Misc,N/A,,,,
-7640b59c-0692-4736-90a1-52a9f5f2d089,,,Pulses-Normal,,Pulses-Normal,,,,Misc,N/A,,,,
+735dc19f-78f0-4f0f-861b-e2b4328521b4,,,AKA,ការវះកាត់យកជើងខាងលើជង្គង់ចេញ,AKA,ការវះកាត់យកជើងខាងលើជង្គង់ចេញ,,,Misc,N/A,,,,
+bb9a023b-a077-4aad-baac-021c8ed1f626,,,BKA,ការវះកាត់យកជើងខាងក្រោមជង្គង់ចេញ,BKA,ការវះកាត់យកជើងខាងក្រោមជង្គង់ចេញ,,,Misc,N/A,,,,
+81318009-e9fb-4b8a-9bf6-17082cefd19e,,,Symptom of PN-Pain,រោគសញ្ញាជំងឺសរសៃប្រសាទឈឺ,Symptom of PN-Pain,រោគសញ្ញាជំងឺសរសៃប្រសាទឈឺ,,,Misc,N/A,,,,
+be0c7342-12da-48c1-a247-89379f80e18d,,,Symptom of PN-Pins and Needle,រោគសញ្ញាជំងឺសរសៃប្រសាទឈឺដូចម្ជុលចាក់,Symptom of PN-Pins and Needle,រោគសញ្ញាជំងឺសរសៃប្រសាទឈឺដូចម្ជុលចាក់,,,Misc,N/A,,,,
+4ff6c3ea-2be8-4aac-b371-faa3bb450255,,,Symptom of PN-Numbness,រោគសញ្ញាជំងឺសរសៃប្រសាទស្ពឹកៗ,Symptom of PN-Numbness,រោគសញ្ញាជំងឺសរសៃប្រសាទស្ពឹកៗ,,,Misc,N/A,,,,
+8c001d6a-9525-4008-8d06-eebc713f5d8c,,,Symptom of PN-Burning,រោគសញ្ញាជំងឺសរសៃប្រសាទក្តៅដូចរលាក,Symptom of PN-Burning,រោគសញ្ញាជំងឺសរសៃប្រសាទក្តៅដូចរលាក,,,Misc,N/A,,,,
+0571152b-1f18-4603-9c86-d77883525e61,,,Symptom PAD-Yes,មានឈឺពេលសម្រាក,Symptom PAD-Yes,មានឈឺពេលសម្រាក,,,Misc,N/A,,,,
+c9cfaaf9-0f14-4805-a151-b0544cbd8870,,,Symptom of PAD-No,អត់មានឈឺពេលសម្រាក,Symptom of PAD-No,អត់មានឈឺពេលសម្រាក,,,Misc,N/A,,,,
+7640b59c-0692-4736-90a1-52a9f5f2d089,,,Pulses-Normal,ជីពចរជើងធម្មតា,Pulses-Normal,ជីពចរជើងធម្មតា,,,Misc,N/A,,,,
 bbf9ee15-e3df-4026-825f-e17325027795,,,Pulse-Abnormal,,Pulse-Abnormal,,,,Misc,N/A,,,,
```

Comment:
@kdaud this is missing please add it from [here](https://docs.google.com/spreadsheets/d/13PHkzorfTmYRjuIJv3fCml3zBAuDba6Mgdpu4H7cAg8/edit#gid=0&range=188:188)

`Pulse Abnormal`

Reply:
ជីពចរជើងធម្មតា